src/components/tab-group/bl-tab-group.css

```
@@ -7,12 +7,11 @@
 
 .tabs-list {
   overflow-x: scroll;
-
-  /* FIXME: use variables */
-  border-radius: 6px 6px 0 0;
+  border-radius: var(--bl-border-radius-m) var(--bl-border-radius-m) var(--bl-border-radius-0)
+    var(--bl-border-radius-0);
 }
 
 .panels {
-  /* FIXME: use variables */
-  border-radius: 0 0 6px 6px;
+  border-radius: var(--bl-border-radius-0) var(--bl-border-radius-0) var(--bl-border-radius-m)
```

Need to remove `--bl-border-radius-0` variable. 
```suggestion
  border-radius: 0 0 var(--bl-border-radius-m)
```